csharp_dotnetcore/Using-Cards/Using_Cards.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
We don't normally check in the .sln file unless it's a multi-project project. In this case, I think this file can be removed, as the only project is the Using_Cards.csproj.

Reply:
I'm still seeing the .sln file in the PR. Was it removed?